dashboard/src/app/admin/user-management/user-management.controller.js

```diff
@@ -67,23 +71,48 @@ export class AdminsUserManagementCtrl {
   }
 
   /**
-   * Load more factories, coll this function on list scroll down
+   * Ask for loading the users page in asynchronous way
+   * @param pageKey - the key of page
    */
-  loadNextPage() {
-    this.skipCount = this.users.length;
-
-    let promise = this.codenvyAPI.getUser().fetchUsers(this.maxItems, this.skipCount);
+  fetchUsersPage(pageKey) {
+    let promise = this.codenvyAPI.getUser().fetchUsersPage(pageKey);
 
     promise.then(() => {
       this.updateUsers();
     }, (error) => {
-      if (error.status !== 304) {
-        this.cheNotification.showError(error.data.message ? error.data.message : 'Update information failed.');
+      if (error.status === 304) {
+        this.updateUsers();
+      } else {
+        this.cheNotification.showError(error.data && error.data.message ? error.data.message : 'Update information failed.');
       }
     });
   }
 
   /**
+   * Returns true if the next page is exist.
+   * @returns {boolean}
+   */
+  noNextPage() {
+    return this.pagesInfo.currentPageNumber === this.pagesInfo.countOfPages;
+  }
+
+  /**
+   * Returns true if the previous page is exist.
+   * @returns {boolean}
+   */
+  noPreviousPage() {
```

Comment:
noPreviousPage -> hasPreviousPage, then "Returns true if the previous page exists."